### ISSUE TITLE
When will be available a new official release?

### ISSUE THREAD

**Issue by matteobaccan** [highsource/w3c-schemas#24]
Hi

when the final 1.4.1 or a new 1.5 release will be published?

all the best
matteo baccan

**Comment by soc** [comment]
@matteobaccan I'm currently trying to roll a new release over in https://github.com/w3c-schemas/w3c-schemas. Let me know if you have changes that should be merged!

**Comment by matteobaccan** [comment]
Hi

my changes are on branch issue23
I'm confused: you are not the original developer of the library: It's correct?

ciao
matteo

**Comment by soc** [comment]
Hi Matteo,

sadly not, see https://github.com/highsource/ogc-schemas/issues/225 for the background.

Thanks,

Simon

**Comment by matteobaccan** [comment]
Hi soc

I have read. Horrible news.

I have some projects with Alexey code.
This is one of them. 

I have seen your work: you don't have merged in master the branch issue-23. I think this branch includes all new patch to master.

ciao
matteo

**Comment by soc** [comment]
No, I haven't merged that yet, I was focused on making the code build and deploy first.

**Comment by matteobaccan** [comment]
OK

I have added a watch on the fork. 
I'll check new releases and I'll try to integrate them into my code

Good job

ciao
matteo

**Comment by soc** [comment]
You could also create a PR for #23 if you like and I'll try to understand and merge it!

**Comment by soc** [comment]
Or I could just add you to the [https://github.com/w3c-schemas](https://github.com/w3c-schemas) organization, if you like?

**Comment by matteobaccan** [comment]
Ooo yes, add me :)

**Comment by soc** [comment]
@matteobaccan you should have received an invitation by mail, let me know if you are missing privileges for the repository, not sure what's the defaults.

**Comment by matteobaccan** [comment]
Hi

I have accepted the invitation. At the moment all works fine.

ciao
matteo
